src/main/java/pl/tomaszdziurko/jvm_bloggers/mailing/LogMailPostAction.java

```diff
@@ -0,0 +1,39 @@
+package pl.tomaszdziurko.jvm_bloggers.mailing;
+
+import org.springframework.context.annotation.Profile;
+import org.springframework.stereotype.Component;
+
+import java.util.concurrent.CountDownLatch;
+
+/**
+ * This class is intended to use in tests only and probably should be moved from here. <br/>
+ * The main purpose of this class is to act as synchronisation barrier between multiple threads. <br/>
+ * The very perfect usage for that is when emails was sent and we want to know how many.
+ *
+ * @see pl.tomaszdziurko.jvm_bloggers.mailing.LogMailSender
+ * @author Adam Dec
+ * @since 0.7.0
+ */
+@Component
+@Profile({"dev", "test"})
+public class LogMailPostAction implements MailPostAction {
```

Comment:
I don't really get why we need this class and this inferface at all :) Care to elaborate? :)